rules/integrations/aws/credential_access_aws_getpassword_for_ec2_instance.toml

```
@@ -0,0 +1,108 @@
+[metadata]
+creation_date = "2024/04/10"
+integration = ["aws"]
+maturity = "production"
+min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
+min_stack_version = "8.3.0"
+updated_date = "2024/05/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the first occurrence of a user identity in AWS using `GetPassword` for the administrator password of an EC2 instance with an assumed role. Adversaries may use this API call to escalate privileges or move laterally within EC2 instances.
+"""
+from = "now-9m"
+index = ["filebeat-*", "logs-aws.cloudtrail*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "User Identity Retrieving Administrator Credentials from EC2 Instance with an Assumed Role"
```

rule.name looks a bit long:

```suggestion
name = "EC2 Admin Credential Fetch via Assumed Role"
```